src/locales/it.json

```diff
@@ -73,7 +73,12 @@
         },
         "title": "Segnala un errore",
         "cancel": "Annulla",
-        "send": "Invia"
+        "send": "Invia",
+        "confirmation": {
+            "title": "Confirma l'invio",
+            "text": "Sei sicuro di voler inviare la segnalazione?",
+            "warningText": "La segnalazione degli errori non ha lo scopo di ordinare una revisione o una consultazione. Per ordinare, contattare il medico prescelto."
```

Comment:
This probably requires a similar update as English and Slovenian. It may be better to leave it untranslated, to show it in English until it is translated by translators.
```suggestion
```